lib/charms/prometheus_k8s/v0/prometheus_scrape.py

```diff
@@ -1176,16 +1173,6 @@ def _static_scrape_config(self, relation) -> list:
             scrape_configs, hosts, topology
         )
 
-        # If scheme is https but no ca section present, then auto add "insecure_skip_verify",
-        # otherwise scraping errors out with "x509: certificate signed by unknown authority".
-        # https://prometheus.io/docs/prometheus/latest/configuration/configuration/#tls_config
```

Comment:
Should we document this?

> For https scrape targets we still do not render a `tls_config` section because
> certs are expected to be made available by the charm via the `update-ca-certificates` mechanism.